### DISCUSSION TITLE
fields with conflicting configuration

### DISCUSSION THREAD

**Discussion by karel-debrabandere-bosa-fgov-be** [OP-TED/eForms-SDK#271] [Q&A] [answered]
I'm working in sub-notice-type 16. In that context, I am having the following issues:
- `BT-633-Organization` is being displayed in the form, but according to the xsd it should not be allowed:
  ```
    <xsd:complexType name="OrganizationType">
        <xsd:sequence>
            <xsd:choice minOccurs="0" maxOccurs="1">
                <xsd:sequence>
                    <xsd:element ref="efbc:GroupLeadIndicator" minOccurs="0" maxOccurs="1"/>
                    <xsd:element ref="efbc:AcquiringCPBIndicator" minOccurs="0" maxOccurs="1"/>
                    <xsd:element ref="efbc:AwardingCPBIndicator" minOccurs="0" maxOccurs="1"/>
                </xsd:sequence>
                <xsd:sequence>
                    <xsd:element ref="efbc:ListedOnRegulatedMarketIndicator" minOccurs="0" maxOccurs="1"/>
                    <xsd:element ref="efbc:NaturalPersonIndicator" minOccurs="0" maxOccurs="1"/>
                    <xsd:element ref="efac:UltimateBeneficialOwner" minOccurs="0" maxOccurs="unbounded"/>
                </xsd:sequence>
            </xsd:choice>
            <xsd:element ref="efac:Company" minOccurs="0" maxOccurs="1"/>
            <xsd:element ref="efac:TouchPoint" minOccurs="0" maxOccurs="unbounded"/>
        </xsd:sequence>
    </xsd:complexType>
  ```
- `OPT-140-Lot` is being displayed in the form, but is not mandatory. According to the xsd it should be mandatory:
  ```
  <xsd:complexType name="DocumentReferenceType">
      <xsd:sequence>
         <!-- ... -->
         <xsd:element ref="cbc:ID" minOccurs="1" maxOccurs="1"/>
         <!-- ... -->
      </xsd:sequence>
   </xsd:complexType>
  ```

**Comment by YvesJo** [comment, marked as answer]
Some rules aren't currently applied on the UI side leading to display of fields. BT-633-Organization (Natural Person Indicator) belongs to a set of fields for Economic Operators only, also it shall not be used for Buyers.
Regarding OPT-140-Lot, any Document Reference requires an Identifier (cbc:ID), this requirement is expressed in the XSD. This identifier has to exist as soon as further information exists in the group. If the Group does not have to exist itself (i.e. is optional), then the field may not be mandatory. It only becomes mandatory when any further information is provided.
